test/runtests.jl

```diff
@@ -11,8 +11,36 @@ using FilePathsBase.TestPaths
 using UUIDs: uuid4
 using JSON3
 
-aws = AWSConfig()
+if VERSION >= v"1.5"
+    using Minio
+    AWS.aws_account_number(::Minio.MinioConfig) = "123"
 
+    # We run most tests under Minio. This can be done locally by those
+    # without access to the s3 bucket under which CI is performed.
+    # We then run all tests with s3 directly.
+
+    # We use multiple directories so that Minio can support versioning.
+    dirs = [mktempdir() for _ =1:12]
```

Comment:
Seems nicer to use a single tempdir with subdirectories if you ever need to do some manual debugging:
```suggestion
    root = mktempdir()
    dirs = [mkdir(joinpath(root, string(i))) for i in 1:12]
```